ssi-jwt/src/lib.rs

```diff
@@ -16,7 +16,6 @@ pub fn encode_sign<Claims: Serialize>(
     let header = Header {
         algorithm,
         key_id: key.key_id.clone(),
-        type_: Some("JWT".to_string()),
```

Comment:
Not sure why this was done. Test without deletion.

Reply:
This is the one change made on the "modify-encode-sign" branch before the RSS work started.